collections/_article/dev-snapshot-godot-4-2-beta-1.md

```diff
@@ -0,0 +1,346 @@
+
+Smaller but notable feature additions have made their way into the release as well. You can now load OGG files at runtime, from a buffer or from a file path ([GH-78084](https://github.com/godotengine/godot/pull/78084) by [K. S. Ernest Lee](https://github.com/fire)). Same goes for SVG files, which can be loaded from a binary buffer or a string ([GH-78248](https://github.com/godotengine/godot/pull/78248) by [Felipe Augusto Marques](https://github.com/felaugmar)).
+
+![GodSVG is an SVG editor app made with Godot 4 by MewPurPur](/storage/blog/godot-4-2-beta/svg-editor-by-mewpurpur.webp)
+
+_[GodSVG](https://github.com/MewPurPur/GodSVG) is an SVG editor app made with Godot 4 by [MewPurPur](https://github.com/MewPurPur)._
+
+
+### 2D and 3D features
+
+Godot is a general purpose game engine, and features a variety of components aimed at helping you develop both 2D and 3D games. While not necessarily core, they are an integral part of the engine and thus get love an attention all the same in every release.
+
+If you are developing 2D games, two new features deserve your attention. First of all, [Riteo](https://github.com/Riteo) takes a break from working on Wayland support (coming soon!) and implements forced integer scaling ([GH-75784](https://github.com/godotengine/godot/pull/75784)). Enabled with a project setting integer scaling ensures that no matter the aspect ratio you get a square pixel grid without distortions. Another useful tool under your belt is the new `closed` property for the `Line2D` nodes, which allows you to draw enclosed lines with uninterrupted visuals ([GH-79182](https://github.com/godotengine/godot/pull/79182) by [MewPurPur](https://github.com/MewPurPur)).
+
+For projects that require precise normals from contact points of raycasts, the ability to get barycentric coordinates is going to be invaluable ([GH-71233](https://github.com/godotengine/godot/pull/71233)). Whether you want to make an F-Zero-like game, or implement gravity-defying walks on walls, you can thank [PrecisionRender](https://github.com/PrecisionRender) for this new feature. And to help you handle rotation and angles correctly there are now two new global methods, `rotate_toward` and `angle_difference`, courtesy of [etti](https://github.com/ettiSurreal) ([GH-80225](https://github.com/godotengine/godot/pull/80225)).
+
+#### Animations
+
+The animation team is hard at work improving robustness of the animation system, and their biggest contribution to this release by far is an internal rework of both `AnimationPlayer` and `AnimationTree`, unifying parts of their implementation under the new `AnimationMixer` node ([GH-80813](https://github.com/godotengine/godot/pull/80813)). This change allows them to address a large number of issues and discrepancies between the two nodes. And I think [Tokage](https://github.com/TokageItLab) has snuck in a feature or two with it as well!
```

Comment:
Would be nice to be a bit more specific about the "feature or two", or just not mention it at all, rather than this sneaky mention.